src/MCPClient/lib/clientScripts/normalizeReport.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python2
+
+from __future__ import print_function
+
+from argparse import ArgumentParser
+import logging
+from smtplib import SMTPException
+from os.path import basename
```

Comment:
Two nitpicks: I try to keep the import list sorted (ie os before smtplib), and I usually `import os.path` so when I use `os.path.basename` it's clear where it comes from, and not a locally-defined basename.